packages/now-static-build/test/fixtures/30-jekyll/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "scripts": {
+    "build": "jekyll build && mv _site public"
```

Comment:
@leo The default jekyll output folder is `_site`. This can be changed by adding a line to `_config.yml` but I would imagine most people don't change it.

Do you want the test to use `mv` or set `_config.yml` to the following?

```yml
destination: _site
```

Reply:
@styfle maybe we can select the framework when a Gemfile was found, and then define the output dir there, like for the other frameworks.

Reply:
@AndyBitz That was what I started implementing at first and the @leo said to hold off on that part. The problem right now is that zero config only works with package.json. So we first need to make it work without package.json before we can get into reading the Gemfile and doing specific framework changes 👍

Reply:
Indeed. In the future, the whole system around zero config detection will be **more generic**. In turn, we already want to start here by only doing two things for Jekyll (that also apply to any other framework):

- If there's a `Gemfile`, install the dependencies
- Extend the build image to include what's necessary